doc/en/reference.rst

```diff
@@ -185,6 +185,11 @@ Mark a test function as using the given fixture names.
     This mark has no effect when applied
     to a **fixture** function.
 
+.. note::
+   When using `usefixtures` in hooks, it can only load fixtures when applied to a **test** before test setup.
+   To modify fixtures after test setup, append the fixture name to item.fixturenames.
```

Comment:
I'm not so sure about this last phrase though, depending the moment when `item.fixturenames` is changed, it might not have any effect anymore. Perhaps it is best to remove it altogether to avoid confusion?